dev-aws/kafka-shared/dev-enablement/proximo-examples.tf

```diff
@@ -0,0 +1,23 @@
+resource "kafka_topic" "proximo_example" {
+  name               = "dev-enablement.proximo-example"
+  replication_factor = 3
+  partitions         = 10
```

Comment:
Probably 3 partitions would be enough for this test, right?

Reply:
For sure, [appplied ](https://github.com/utilitywarehouse/kafka-cluster-config/pull/234/commits/bbff0533a4cf54ad553b0bf96b574d9a4bcabb91). I copied the configuration from pubsub examples- I wonder why do we need so much of redundancy for such simple services. 
I guess we could also reduce the replication factor, but since the topic will contain minimal amount of data and this grants additional reliability, I'd just go with defaults